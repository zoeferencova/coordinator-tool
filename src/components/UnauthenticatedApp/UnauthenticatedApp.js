import React from 'react';
import { Routes, Route } from 'react-router-dom';
import LandingPage from '../../routes/LandingPage/LandingPage';
import LoginPage from '../../routes/LoginPage/LoginPage';
import RegistrationPage from '../../routes/RegistrationPage/RegistrationPage';
import NotFoundPage from '../../routes/NotFoundPage/NotFoundPage';

const UnauthenticatedApp = ({ setLoggedIn }) => (
  <main className='App'>
    <Routes>
      <Route exact path={'/'} element={<LandingPage setLoggedIn={setLoggedIn} />} />
      <Route path={'/login'} element={<LoginPage setLoggedIn={setLoggedIn} />} />
      <Route path={'/register'} element={<RegistrationPage />} />
      <Route element={NotFoundPage} />
    </Routes>
  </main>
)

export default UnauthenticatedApp;