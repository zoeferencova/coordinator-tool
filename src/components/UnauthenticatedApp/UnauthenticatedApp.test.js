import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UnauthenticatedApp from './UnauthenticatedApp'

describe(`UnauthenticatedApp component`, () => {
  it('renders a .UnauthenticatedApp by default', () => {
    const wrapper = shallow(<UnauthenticatedApp />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})