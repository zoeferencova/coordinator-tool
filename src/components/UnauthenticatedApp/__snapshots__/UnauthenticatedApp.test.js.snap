// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnauthenticatedApp component renders a .UnauthenticatedApp by default 1`] = `
<main
  className="App"
>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/login"
    />
    <Route
      component={[Function]}
      path="/register"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</main>
`;
